book/wildfires/sensors/wildfires-sensors-modis.ipynb

```diff
@@ -0,0 +1,589 @@
+{
```

Comment:
<p><code>pyspectral</code> isn't in the default environment so we should install it first:</p><pre class="ql-syntax" spellcheck="false">!pip -q install pyspectral
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/acocac/environmental-ai-book/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>